dist/.github1/elixir.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
I assume `.github1` is a temporary directory?

Reply:
Nope. This `dist` stuff is not handled "by hand". It's generated with `npm run build`. I also found this odd (and asked around for a better explanation), but couldn't get one or find one. I just trust it `ncc` knows what it's doing and that if a fix comes around we'll benefit from it, in the future.